Workflow/Tests/SubtreeManagerTests.swift

```diff
@@ -140,163 +140,6 @@ final class SubtreeManagerTests: XCTestCase {
-        struct WorkerWorkflow: Workflow {
-            var startExpectation: XCTestExpectation
-            var endExpectation: XCTestExpectation
-
-            enum State {
-                case notWorking
-                case working
-            }
-
-            func makeInitialState() -> WorkerWorkflow.State {
-                return .notWorking
-            }
-
-            func render(state: WorkerWorkflow.State, context: RenderContext<WorkerWorkflow>) -> Bool {
-                switch state {
-                case .notWorking:
-                    return false
-                case .working:
-                    context.awaitResult(
-                        for: ExpectingWorker(
-                            startExpectation: startExpectation,
-                            endExpectation: endExpectation
-                        ),
-                        outputMap: { output -> AnyWorkflowAction<WorkerWorkflow> in
-                            AnyWorkflowAction.noAction
-                        }
-                    )
-                    return true
-                }
-            }
-
-            struct ExpectingWorker: Worker {
-                var startExpectation: XCTestExpectation
-                var endExpectation: XCTestExpectation
-
-                typealias Output = Void
-
-                func run() -> SignalProducer<Void, Never> {
-                    return SignalProducer<Void, Never>({ [weak startExpectation, weak endExpectation] observer, lifetime in
-                        lifetime.observeEnded {
-                            endExpectation?.fulfill()
-                        }
-
-                        startExpectation?.fulfill()
-                    })
-                }
-
-                func isEquivalent(to otherWorker: WorkerWorkflow.ExpectingWorker) -> Bool {
-                    return true
-                }
-            }
-        }
-
-        let startExpectation = XCTestExpectation()
-        let endExpectation = XCTestExpectation()
-        let manager = WorkflowNode<WorkerWorkflow>.SubtreeManager()
-
-        let isRunning = manager.render { context -> Bool in
-            WorkerWorkflow(
-                startExpectation: startExpectation,
-                endExpectation: endExpectation
-            )
-            .render(
-                state: .working,
-                context: context
-            )
-        }
-
-        XCTAssertEqual(true, isRunning)
-        wait(for: [startExpectation], timeout: 1.0)
-
-        let isStillRunning = manager.render { context -> Bool in
-            WorkerWorkflow(
-                startExpectation: startExpectation,
-                endExpectation: endExpectation
-            )
-            .render(
-                state: .notWorking,
-                context: context
-            )
-        }
-
-        XCTAssertFalse(isStillRunning)
-        wait(for: [endExpectation], timeout: 1.0)
-    }
-
-    func test_subscriptionsUnsubscribe() {
```

Comment:
We should test this still too in `WorkflowReactiveSwitftTests`

Reply:
I wasn't sure if this was left over from the old `Subscriptions` world.

Reply:
I'm not sure if we're testing anything relevant here. Dropping this one unless you believe it does add value.

Reply:
Yeah, I guess it currently ends up being a test of SignalWorker, which should have its own independent test coverage.